csrc/scheduler/no_op.cpp

```diff
@@ -30,8 +30,10 @@ NoOpScheduler::NoOpScheduler(
 
 namespace {
 bool allOutputsArePointerArithmetics(Fusion* fusion) {
-  const AliasAnalysisResult analysis =
-      findAliases(fusion, /*can_override_empty_allocation_domain=*/false);
+  const AliasAnalysisResult analysis = findAliases(
+      fusion,
+      /*can_override_empty_allocation_domain=*/false,
+      /*can_alias_intermediate=*/false);
```

Comment:
```suggestion
      /*allow_output_alias_intermediate=*/false);
```

Side note: Apparently, we haven't enabled https://clang.llvm.org/extra/clang-tidy/checks/readability/named-parameter.html; otherwise you'll see a clang-tidy error. I'll try to add that to our lint config.

Reply:
Weird -- I no longer understand why this didn't trigger a clang-tidy error. 

Although readability-named-paraemter is not turned on, bugprone-argument-comment is. It serves the same purpose, and indeed kicks in when I mistyped the argument name: https://github.com/NVIDIA/Fuser/pull/2626 and https://github.com/NVIDIA/Fuser/actions/runs/9982268422/job/27587668042?pr=2626.